tests/instructions/syntax.txt

```diff
@@ -1,6 +1,5 @@
-# Quick syntax check (2 minutes on 16-core server)
+# Relatively quick syntax check (20 minutes on 16-core server)
```

Comment:
20 minutes is how long it took for me (on master branch without this PR). Did clang get 10x slower in 5 years? :o (More than 10x because computers get faster. But less because the code got bigger. Let's guess that these two effects roughly cancel out.) I heard that llvm gets slower with each version, but would have expected less than 1.6x/year.